### PR TITLE
add examples for different resolveMode options.

### PR DESCRIPTION
**Pull request by zoellner**
Related to #173 (#134)

*Description of changes:*
Add documentation for reolveMode options `PROMISE` and `CALLBACK`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by zoellner** [comment]
Note that this PR contains another commit since I am working against the master branch as noted in CONTRIBUTING and making the PR against the develop branch as noted in the PULL_REQUEST_TEMPLATE

**Comment by brettstack** [comment]
Sorry, that contributing guide is incorrect (default/standard from AWS). I'll change it to request changes are made based on develop branch.

File: README.md [hunk 1]
```diff
@@ -44,6 +46,39 @@ app.get('/', (req, res) => {
...
+'use strict'
+const awsServerlessExpress = require('aws-serverless-express')
+const app = require('./app')
+const server = awsServerlessExpress.createServer(app)
+
+// note that callback the promise is always resolved
+// i.e. errors still lead to a resolved promise
+// The resolved promise has a a statusCode property that can be checked for error codes
+
+exports.handler = async (event, context) => { return awsServerlessExpress.proxy(server, event, context, 'PROMISE') }
```
Comment (Craigfis) [inline README.md:62]: the .proxy call should be suffixed with .promise I believe.

Comment (wattry) [reply README.md:62]: If your handler is't doing any more async work why would someone use this feature?

**Pull request merged**